### ISSUE TITLE
External link highlighting vs accessibility

### ISSUE THREAD

**Issue by amoebanath** [localgovdrupal/localgov_theme#214]
https://github.com/localgovdrupal/localgov_theme/blob/364fcab72504b0c560bc5ea2bc72fe3cb51d47e7/assets/js/main.js#L29

We've had a site go through accessibility testing, and had failures because the external link icon 'is not accessible'.

This has opened up two thoughts:
1. Should this code instead be appending to the link text " (opens in a new tab)" 
[GovUK stopped using the icon a few years ago](https://designnotes.blog.gov.uk/2016/11/28/removing-the-external-link-icon-from-gov-uk/)
2. Should this code also be adding `rel="noopener"` attribute [to prevent security risks](https://techblog.topdesk.com/security/developers-need-know-reverse-tabnabbing/)

**Comment by keelanfh** [comment]
Closing as deprecated and this is no longer the case in localgov_base. Some discussion of adding one in https://github.com/localgovdrupal/localgov_base/issues/53
